lib/screens/settings.dart

```diff
@@ -10,6 +10,8 @@ import 'package:fogosmobile/constants/endpoints.dart';
 import 'package:fogosmobile/models/app_state.dart';
 import 'package:fogosmobile/actions/preferences_actions.dart';
 
+import 'components/fire_gradient_app_bar.dart';
```

Comment:
i don't have a preference regarding referencing internal dependencies with a relative path (line 13) or absolute path (line 11), but they should be consistent

Reply:
I agree with this. Missed it somehow.

Reply:
forgot to mention that this occurs on other files as well